src/generator/AutoRest.Go/rakefile.rb

```diff
@@ -0,0 +1,231 @@
+require 'fileutils'
+
+AUTOREST     = "../../core/AutoRest/bin/Debug/net451/win7-x64/AutoRest.exe"
```

Comment:
That file, as far as I understand, when run it regenerates the ARM stuff for the SDK